bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -0,0 +1,39 @@
+{% extends "bare.html" %}
+{% stylesheet 'firefox_new_firstrun' %}
+
+{% block content %}
+    <div>
+        <div id="scene">
+            <div class="fxaccounts-container">
+                <h2 id="title"> Already Using Firefox?</h2>
+                <div id="left-divider">
+                    <p>Sign in to your account and we'll sync the bookmarks, passwords and other great things you've saved to Firefox on other devices.</p>
+                    <div>
+                      <img src="/media/img/firefox/firstrun-horizon/sample.png">
+                    </div>
+                    <a href = "https://www.mozilla.org/en-US/firefox/sync/"> Learn more about Firefox Accounts</a>
+                </div>
+              <div class="fxaccounts" id="fxa-iframe-config" data-host="{{ settings.FXA_IFRAME_SRC }}" data-mozillaonline-host="{{ settings.FXA_IFRAME_SRC_MOZILLAONLINE }}">
+                  <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}signin?utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=firstrun&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
+                  <button id="skip-button"> Skip This Step</button>
+              </div>
+          </div>
+
+            <img id="night-sky" src="/media/img/firefox/firstrun-horizon/night-scene/sky.png">
+            <img id="sky" src="/media/img/firefox/firstrun-horizon/day-scene/sky.png">
```

Comment:
yes, I will remove all of the x2 pngs
I suppose it's arguable, but I was working under the impression that SVGs are less efficient, especially when animating.